src/Sarif/Core/Artifact.cs

```diff
@@ -40,13 +40,13 @@ public partial class Artifact : ISarifNode
             // occurred. Something to discuss moving forward.
             try
             {
-                bool workTodo = dataToInsert.HasFlag(OptionallyEmittedData.Hashes)    ||
```

Comment:
>     [](start = 82, length = 4)

As you can see, I use whitespace often to line up boolean operators for better readability. Wonder if we can dial this one back. My care factor is low, though. :)